### DISCUSSION TITLE
Should there be a separate bucket for all the Sordum apps?

### DISCUSSION THREAD

**Discussion by kris3713** [ScoopInstaller/Extras#9370] [General]
On the website sordum.org, they have 63 apps in total that a lot of Windows 11 and 10 users make use of every day. Should these apps continued to be added to the `extras` bucket, or should they have their very own bucket called `sordum`?

**Comment by rasa** [comment]
I have always supported creating separate buckets, to allow us to easily distribute the administrative burden. But now that we allow subdirectories in our bucket directory, one of our pain points has been removed.

**Reply by kris3713** [reply]
So could I make a PR where I add a folder called `sordum` to the **bucket** folder that contains all the apps made by Sordum?

**Comment by rasa** [comment]
Note that only a subset of their apps are identified as freeware: https://www.sordum.org/category/our-freewares/

**Reply by kris3713** [reply]
Oh so that means that there are only ~~45~~ apps that Sordum offers for free.


EDIT: I meant to say 35 Free apps.

**Comment by goyalyashpal** [comment]
can this discussion be closed?
